app/packages/operators/src/state.ts

```diff
@@ -805,7 +802,14 @@ export function useOperatorPlacements(place: Place) {
       filters,
       selectedSamples,
     });
-  }, [datasetName, view, extendedStages, filters, selectedSamples]);
+  }, [
+    setThrottledContext,
+    datasetName,
+    view,
+    extendedStages,
+    filters,
+    selectedSamples,
```

Comment:
Can `selectedLabels` be added here? It seems like the fix for https://github.com/voxel51/fiftyone/issues/3740

Reply:
Yah. However, it would not fix #3740 by adding here as this context is only used for resolving placement. 

I will look into #3740